_posts/2020-06-11-trusted-web-activities.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Trusted Web Activities
+date: 2020-06-11
+tags: tutorial pwa PWA webapps spa android react
+author: Simon Bracegirdle
+image: /img/pwa.png
+---
+
+<center><img src="/img/pwa.png" /></center>
+<br/>
+
+One of the promises of building apps on the web is the realisation of cross-platform app development — write once, run anywhere. A single development team can deliver a solution to customers that works on any device or platform they use.
+
+With the rise of Progressive Web Apps over the last few years, that promise is being fulfilled. One of the last remaining hurdles for uptake on mobile devices is distribution of web apps in the native mobile app stores. These stores ease discoverability and installation for users, but have been exclusive to apps built natively for each platform.
+
+There has been a few previous attempts to integrate web apps into native app stores. Mozilla's [Firefox OS](https://en.wikipedia.org/wiki/Firefox_OS) comes to mind, as well as [webOS](https://en.wikipedia.org/wiki/WebOS) and Apple's original intention to focus on web apps instead of native apps. But, up until now there hasn't been an offering on either of the two major platforms — iOS and Android.
+ 
+The time is now ripe for that to change.
+
+In this post we'll learn about a new feature in Android called Trusted Web Activities, which enables publishing of web apps to the Google Play Store. We'll also work through an example of setting one up for an existing web app.
+
+Let's get started!
+
+## Background
+
+### What is a Trusted Web Activity?
+
+In the introduction I mentioned Trusted Web Activities, so what are they?
```

Comment:
>In the introduction I mentioned Trusted Web Activities, so what are they?

Unnecessary - suggest delete.

Your intro is short, and you describe what they are in the next para.